runtime/doc/filetype.txt

```diff
@@ -521,6 +521,8 @@ To use Nvim as a manpager: >
 man.vim will always attempt to reuse the closest man window (above/left) but
 otherwise create a split.
 
+The case sensitivity of completion is controlled by 'fileignorecase'.
```

Comment:
Huh, I never noticed that Vim doesn't use `||` for options. Seems a bit inconsistent to me. Why having 2 kinds of highlights?